synapse/axon.py

```diff
@@ -657,6 +658,14 @@ async def get(self, sha256):
         if not await self.has(sha256):
             raise s_exc.NoSuchFile(mesg='Axon does not contain the requested file.', sha256=s_common.ehex(sha256))
 
+        fhash = s_common.ehex(sha256)
```

Comment:
this stuff (except for the logger call) should be a function

Reply:
maybe even something like getLogExtras() on the cell?